docs_nnx/guides/surgery.ipynb

```diff
@@ -209,7 +213,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In this new model, the sub-modules are renamed from `linear(1|2)` to `layer(1|2)`. Since the pytree structure changed, it's impossible to load the old checkpoint with the new model state structure:"
+    "In this new model, the sub-`Module`s are renamed from `linear(1|2)` to `layer(1|2)`. Since the pytree structure has changed, it is impossible to load the old checkpoint with the new model state structure:"
```

Comment:
"it is impossible to load" -> "it is impossible to directly load"

Reply:
thanks, done!